### DISCUSSION TITLE
Modify Template.ftl

### DISCUSSION THREAD

**Discussion by LoVas2** [keycloakify/keycloakify#82] [Q&A] [answered]
Hello,

Is it possible de replace the Template component with mine ?
When a copy/paste the Template.tsx with changing only the imports path, I got a blank page. 

Thx

**Comment by garronej** [comment, marked as answer]
Hi,  
Yes, it is done [here](https://github.com/InseeFrLab/onyxia-web/tree/main/src/ui/components/KcApp)  
Regards,
